### PR TITLE
default check with working

### PR DESCRIPTION
**Pull request by cool-man-vk**
added some minor changes

### CODE REVIEW

File: wagtail/permission_policies/base.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def user_has_permission(self, user, action):
     def user_has_any_permission(self, user, actions):
         """
         Return whether the given user has permission to perform any of the given actions
-        on some or all instances of this model
+        on some or all instances of this model......
```
Comment (lb-) [inline wagtail/permission_policies/base.py:44]: what is the purpose of this change?

**Comment by cool-man-vk** [comment]
Actually I am new to this ,so to check whether it works or not ,I checked
it and it works fine.

**Comment by lb-** [comment]
no problems - let us know if you had any questions on how to contribute @cool-man-vk
